Microsoft.HealthVault.Fhir.UnitTests/ToFhirTests/PersonItemToFhirTests.cs

```diff
@@ -0,0 +1,248 @@
+        public void WhenPersonItemTransformedToFhir_ThenProffessionalTrainingIsAddedAsQualification()
+        {
+            const string training = "AtoZ Training";
+            PersonItem person = GetSamplePerson();
+            person.ProfessionalTraining = training;
+
+            var practitioner = person.ToFhir();
+
+            Assert.IsTrue(practitioner.Qualification.Any());
+            Assert.AreEqual(training, practitioner.Qualification.First().Code.Text);
+        }
+
+        [TestMethod]
+        public void WhenPersonItemTransformedToFhir_ThenIdIsCopiedToIdentifier()
+        {
+            const string id = "DOC3467";
+            PersonItem person = GetSamplePerson();
+            person.PersonId = id;
+
+            var practitioner = person.ToFhir();
+
+            Assert.IsTrue(practitioner.Identifier.Any());
+            Assert.AreEqual(id, practitioner.Identifier.First().Value);
+        }
+
+        [TestMethod]
+        public void WhenPersonItemTransformedToFhir_ThenContactAddressIsCopiedToAddress()
+        {
+            const string description = "14503 Apt# 102";
+            const bool isPrimary = true;
+            const string street = "SE 140th St";//Required Multi
+            const string city = "DC";//Required
+            const string state = "WA";
+            const string postCode = "98008";//Required
+            const string country = "USA";//Required
+            const string county = "County";
+            PersonItem person = GetSamplePerson();
+            var address = new ItemTypes.Address
+            {
+                Description = description,
+                IsPrimary = isPrimary,
+                City = city,
+                State = state,
+                PostalCode = postCode,
+                Country = country,
+                County = county
+            };
+            address.Street.Add(street);
+            var contactInfo = new ContactInfo();
+            contactInfo.Address.Add(address);
+            person.ContactInformation = contactInfo;
+
+            var practitioner = person.ToFhir();
+
+            Assert.IsTrue(practitioner.Address.Any());
+
+            var fhirAddress = practitioner.Address.First();
+
+            Assert.AreEqual(description, fhirAddress.Text);
+            Assert.AreEqual(street, fhirAddress.Line.First());
+            Assert.AreEqual(city, fhirAddress.City);
+            Assert.AreEqual(state, fhirAddress.State);
+            Assert.AreEqual(postCode, fhirAddress.PostalCode);
+            Assert.AreEqual(country, fhirAddress.Country);
+            Assert.AreEqual(county, fhirAddress.District);
+
+            Assert.IsTrue(fhirAddress.Extension.Any(ext => ext.Url == HealthVaultExtensions.IsPrimary));
+        }
+
+        [TestMethod]
+        public void WhenPersonItemTransformedToFhir_ThenContactPhoneIsCopiedToTelecom()
+        {
+            var description = "Office";
+            var isPrimary = true;
+            var number = "4254485432";
```

Comment:
Use a sample phone number (e.g. 555 something) instead of a random phone number.